docker/pharo/launchpad-start

```diff
@@ -10,7 +10,7 @@ fi
 termination_handler() {
   exit_status=0
   if [ $pid -ne 0 ]; then
-    echo 'SIGTERM received, shutting down the application'
+    echo '\nSIGTERM received, shutting down the application'
```

Comment:
You will need echo -e to properly interpret escape sequences, another alternative is doing this:
```suggestion
    echo
    echo 'SIGTERM received, shutting down the application'
```